stdpopsim/catalog/HelMel/genome_data.py

```diff
@@ -1,37 +1,6 @@
 # File autogenerated from Ensembl REST API. Do not edit.
-#
-# Manually edited. See
-# https://github.com/popsim-consortium/stdpopsim/issues/835#issuecomment-815854055
-#
-# Data source:
-# http://ensembl.lepbase.org/Heliconius_melpomene_melpomene_hmel25/Info/Index
-#
-# Chromosomes lengths from Davey et al. 2017
-#
 data = {
-    "assembly_accession": None,  # No accession number available
-    "assembly_name": "Hmel2.5",
-    "chromosomes": {
-        "1": {"length": 17206585, "synonyms": []},
-        "2": {"length": 9045316, "synonyms": []},
-        "3": {"length": 10541528, "synonyms": []},
-        "4": {"length": 9662098, "synonyms": []},
-        "5": {"length": 9908586, "synonyms": []},
-        "6": {"length": 14054175, "synonyms": []},
-        "7": {"length": 14308859, "synonyms": []},
-        "8": {"length": 9320449, "synonyms": []},
-        "9": {"length": 8708747, "synonyms": []},
-        "10": {"length": 17965481, "synonyms": []},
-        "11": {"length": 11759272, "synonyms": []},
-        "12": {"length": 16327298, "synonyms": []},
-        "13": {"length": 18127314, "synonyms": []},
-        "14": {"length": 9174305, "synonyms": []},
-        "15": {"length": 10235750, "synonyms": []},
-        "16": {"length": 10083215, "synonyms": []},
-        "17": {"length": 14773299, "synonyms": []},
-        "18": {"length": 16803890, "synonyms": []},
-        "19": {"length": 16399344, "synonyms": []},
-        "20": {"length": 14871695, "synonyms": []},
-        "21": {"length": 13359691, "synonyms": []},
-    },
+    "assembly_accession": "GCA_000313835.1",
+    "assembly_name": "Hmel1",
+    "chromosomes": {},
```

Comment:
And another one that dissappeared?

Reply:
Looks like another manually manipulated one

Reply:
Well this one has [no karyotype entry](http://rest.ensembl.org/info/assembly/heliconius_melpomene?synonyms=1):
```
--- 
assembly_accession: GCA_000313835.1
assembly_date: 2012-02
assembly_name: Hmel1
coord_system_versions: 
  - Hmel1
default_coord_system_version: Hmel1
genebuild_initial_release_date: 2012-03
genebuild_last_geneset_update: 2012-03
genebuild_method: import
genebuild_start_date: 2012-03-HGC
golden_path: 273786188
karyotype: []

top_level_region: 
  - 
    coord_system: scaffold
    length: 163478
    name: HE667775
    synonyms: 
      - 
        dbname: INSDC
        name: HE667775.1
```

Reply:
... and, it's just a bunch of scaffolds - nothing as long as in the assembly we've got.